news/2022-08-01-osumonthly-1.md

```diff
@@ -0,0 +1,211 @@
+
+[➥ *Find the forum article here!*](https://osu.ppy.sh/home/news/2022-07-07-osu-mania-4k-world-cup-2022-registrations-now-open) 
+
+
+### ![](/wiki/shared/mode/catch.png) osu!catch
+#### At the beginning of July, the recent eleventh iteration of osu!catch World Cup has been concluded.
+
+For the fourth time in a row and a total of eight wins, South Korea are the osu!catch champions. The roster consisting of JISAN, Rells, YesMyDarknesss, x_angelkawaii_x, DreStar and Byeol. With second and third places placed by United States (Secre, Lexii, Dahcreeper, Zak, Colin, Elux) and China (Dusk, Callionet, Coevir, Crystal, KagamineRin02, Kitakami).
+
+It's not a secret that players from South Korea are so good that they could swap each other and win the entire tournament no matter what's ahead. With no qwhj1027 and Abstract- winning the previous year, some analysts had doubts that they could claim the winning title this year. Thankfully or not, that wasn't the case here at all. With 11 maps lost in this entire tournament, South Korea showed that even with not the strongest players this game could possibly meet there are plenty of players that can still impress in this mode.
+
+United States entered this tournament with almost the same roster with swapping Kisetsunotori to Elux being the only roster change. Although they've fallen to the losers bracket after playing a match against South Korea, United States showed their determination to achieve something worthy. In the end, they've claimed themselfes as grand finalists for the second time in this world cup.
+
+With the recent change of swapping from Group Stages to Qualifiers format, most of the players and viewers didn't expect to see China on a very top despite failing their previous run placing 8th-7th place. Their expectations worth it in the end as they instantly gained their glory reaching winners finals bracket without losing a single match until meeting with South Korea and untimately to United States. Even with a long year absense, China showed that even being absent for an entire year they can still make a good fight with top teams.
+
+There's still plenty of tournaments running though even after the conclusion of CWC! The digit tournaments, 4 Digit Catch World Cup 2022 and 3 Digit Catch World Cup 2022, are up and running and a second iteration of Catch French Dual Cup 2 currently in the registered phase. There's nothing to talk about these tournaments yet but maybe with the new osu!monthly release we can fully recap what was going on with conclusions and stuff!
+
+### ![](/wiki/shared/mode/taiko.png) osu!taiko
+ #### As if we could allow for osu! to take all the glory in its showcasing of top players and their performances, Taiko takes it upon themself to rival even that of *The Roundtable* with their very own annual entry of **[Expert Global Taiko Showdown 2022](https://osu.ppy.sh/wiki/en/Tournaments/GTS/EGTS_2022)**! 
```

Comment:
```suggestion

As if we could allow for osu! to take all the glory in its showcasing of top players and their performances, osu!taiko takes it upon itself to rival even that of *The Roundtable* with their very own annual entry of **[Expert Global Taiko Showdown 2022](https://osu.ppy.sh/wiki/en/Tournaments/GTS/EGTS_2022)**!
```
I'm very sure that you mean the game mode. In this case, I would suggest to write that the taiko community is also engaged in tournaments and is making efforts in this regard.

Reply:
Yep!